Automattic/wp-calypso
# Fix/spt sidebar open by default

#### Changes proposed in this Pull Request

* Adds a check to force the SPT "Page Layout" Sidebar plugin to open by default.

I will add a followup PR to core later this week to add an `initialOpen` property to the `PluginDocumentSettingPanel` component.  This component uses a `PanelBody` component, and the `PanelBody` component has this prop available and designed into its structure so it seems to make sense that the `PluginDocumentSettingPanel` should be updated to allow for this optional prop to pass through to the `PanelBody`.

#### Testing instructions


Tested Locally and on Sandbox
* Run this PR.
* Navigate to the page editor.
* Verify that the "Page Layouts" sidebar plugin is open.
* Close it.
* Reload or navigate back to the page editor again.
* Verify that it is open by default once more.

Fixes #37781
